--- InstallationsController.php.org	2017-11-23 09:01:03.000000000 +0000
+++ InstallationsController.php	2017-11-23 09:01:07.000000000 +0000
@@ -494,6 +494,11 @@ class InstallationsController extends Ap
 		$data['encoding'] = $installationData['dbEncoding'];
 		$data['dataPattern'] = $installationData['dbDataPattern'];
 		$data['persistent'] = false;
+		$data['flags'] = [
+			PDO::MYSQL_ATTR_SSL_CA => '/usr/local/etc/BaltimoreCyberTrustRoot.crt.pem',
+			PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT => false,
+		];
+
 		return $data;
 	}