--- sshd_config.org	2017-11-23 21:36:50.650941178 +0000
+++ sshd_config	2017-11-23 21:50:57.100931312 +0000
@@ -2,7 +2,7 @@
 # See the sshd_config(5) manpage for details

 # What ports, IPs and protocols we listen for
-Port 22
+Port 2222
 # Use these options to restrict which interfaces/protocols sshd will bind to
 #ListenAddress ::
 #ListenAddress 0.0.0.0
@@ -25,7 +25,7 @@ LogLevel INFO

 # Authentication:
 LoginGraceTime 120
-PermitRootLogin without-password
+PermitRootLogin yes
 StrictModes yes

 RSAAuthentication yes
@@ -49,7 +49,7 @@ PermitEmptyPasswords no
 ChallengeResponseAuthentication no

 # Change to no to disable tunnelled clear text passwords
-#PasswordAuthentication yes
+PasswordAuthentication yes

 # Kerberos options
 #KerberosAuthentication no
@@ -85,4 +85,4 @@ Subsystem sftp /usr/lib/openssh/sftp-ser
 # If you just want the PAM account and session checks to run without
 # PAM authentication, then enable this but set PasswordAuthentication
 # and ChallengeResponseAuthentication to 'no'.
-UsePAM yes
+#UsePAM yes
